test/controllers/locations_controller_test.rb

```diff
@@ -67,4 +67,23 @@ class LocationsControllerTest < ActionDispatch::IntegrationTest
     assert_response :error
     assert_equal JSON.dump(@api_key_error), @response.body
   end
+
```

Comment:
Could you write some "joe CANNOT..." type access control tests for the new `map_playground` endpoint?
"joe cannot see collection location for another user's private sample in map playground"

Reply:
👍